dev/integration_tests/ios_app_with_extensions/pubspec.yaml

```
@@ -13,7 +13,7 @@ name: ios_app_with_extensions
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.12.1"
```

Let's use the recommended version constraint https://dart.dev/null-safety#constraints
```suggestion
  sdk: ">=2.12.0 <3.0.0"
```

---

Thanks @jmagman